.percy.yml

```diff
@@ -0,0 +1,11 @@
+version: 1
+snapshot:
+  widths: [512, 1200, 1634]
```

Comment:
We have at the moment 3 main breakpoints:
- mobile (both navs collapsed)
- tablet (only main nav collapsed)
- desktop (normal view)

The large desktop (4 columns) has been fully implemented yet.

As for the choice of widths, it's just some numbers to be able to trigger the breakpoints.
However, I just kept one size now because, as mentioned, the snapshots taken are kind of broken. We need to look into that. We can have a chat tomorrow in case.

Reply:
I might have found the issue

Reply:
Much better now! 🎉
https://percy.io/commercetools-GmbH/docs-kit-smoke-tests/builds/3048313

Reply:
FYI: this was the issue https://github.com/commercetools/commercetools-docs-kit/pull/22/commits/60a0eba2856800bc1d8cdcfe4062fe2b5ff06a73

Reply:
OK, let's add the wider variant once it's implemented.  Thanks for digging through to find that rem / px thing. 

The big shots are very high due to the long navigation but I tend to split the site into two of which one is for the API related anyways so that's a temporary thing.

Reply:
Yeah, we need to restructure a bit the site navigation and pages.

The px/rem thing was kind of unexpected, but I guess for dimensions is fine to use pixels only.